build.gradle

```diff
@@ -321,6 +321,33 @@ test {
             '-XX:+HeapDumpOnOutOfMemoryError'
     ]
 
+    filter {
+        //include specific method in any of the tests
+        excludeTestsMatching("*Curator*")
```

Comment:
Does this match the fqcn? If so, this might be the point at which we actually namespace our tests properly and just filter on it.

Reply:
Yes. It should understand the same patterns as `--tests` filter. https://docs.gradle.org/current/userguide/java_testing.html#test_filtering